orgs.js

```diff
@@ -1,5 +1,7 @@
 
     var orgs = [
+        {"name": "ibm-watson-iot",
+         "type": "repo"},
```

Comment:
type should be 'org' here, since you want the entire https://github.com/ibm-watson-iot org to be added, not just a single repository

Reply:
OK. That was a typo.